src/styles/theme.js

```diff
@@ -93,6 +93,7 @@ export default ({ color, backgroundColor, borderColor, cardColor }: Props) => ({
     fontSize: 15,
     color: BRAND_COLOR,
     textAlign: 'right',
+    flex: 0.5,
```

Comment:
That means roughly 'half the width of other elements'
This is unlikely to be a good idea for a general-purpose style that will be used in different places.

Reply:
got it, but can I use ```flex: 0.5```  just in the SettingsCard, instead of setting a width